src/micromamba/install.cpp

```diff
@@ -93,6 +93,13 @@ set_install_command(CLI::App* subcom)
         parse_file_options();
         load_configuration();
 
+
+        if (true)
+        {
+            Context::instance().debug_print();
+            exit(0);
```

Comment:
Would be better to return instead of using `exit` to avoid exiting the R or Python interactive shell when using directly libmamba shared lib